src/snackbar.jsx

```diff
@@ -162,7 +161,7 @@ const Snackbar = React.createClass({
 
       //Only Bind clickaway after transition finishes
       this.timerTransitionId = setTimeout(() => {
-        this._bindClickAway();
+        this.timerTransitionId = 0;
```

Comment:
Yeah we can set it to false. That's fine. It's not there to protect from clearing timeout, it just is what I'm using to distinguish that the time out completed. 

On line 227, if the timeout isn't completed, it will not close the snack bar. It uses this value to determine that.